idunn/blocks/wikipedia.py

```diff
@@ -265,6 +289,8 @@ def get_wiki_index(cls, lang):
 
     @classmethod
     def init_wiki_es(cls):
+        if cls._prom_tracker is None:
```

Comment:
I don't think building a `PrometheusTracker` and keeping it as a class variable is useful.

I tihnk we can either:

* remove the PrometheusTracker class, and make all the function module function
we will then use it as `prometheus.exception("RequestException")`

* Instantiate a PrometheusTracker in the prometheus module
we will use it as `prometheus.tracker.exception("RequestException")`

I'd vote for the first solution, but the second one is fine for me too

Reply:
Ok I go for the first one then